src/components/Sing/ScoreSequencer.vue

```diff
@@ -1244,7 +1244,9 @@ const playheadPositionChangeListener = (position: number) => {
   // オートスクロール
   const sequencerBodyElement = sequencerBody.value;
   if (!sequencerBodyElement) {
-    throw new Error("sequencerBodyElement is null.");
+    // HMR時にここにたどり着くことがあるので、ここは警告だけにする
+    warn("sequencerBodyElement is null.");
+    return;
   }
```

Comment:
これは意図しない挙動が起きてるので、throw errorで良い気がしています。
というのと、例えばテストではエラーになってほしい気がするから開発時だけwarnにしたいけど、まあ面倒なのでthrowで良いかな、みたいな気持ちです！

```suggestion
  if (!sequencerBodyElement) {
    throw new Error("sequencerBodyElement is null.");
  }
```

（ここからはただのコメントです）
そもそもこのコンポーネント、HMRするといろいろぶっ壊れますね。。。。
onActivate周りかな。。😇

Reply:
HMRするとデータ準備中のポップアップが外れなくなるんですよね、そのせいで定期的にリロードを強いられちゃうのでこうしてました。

Reply:
あ～～なるほどです。
経験上、HMRで挙動が変になるコンポーネントはなにかおかしいんですよね。unmounted書いてないとか。（これは書いてますが）

開発時warnが良さそう！
お手数おかけしちゃうのですが、FIXMEも書くのお願いしても良いでしょうか 🙇 

`// FIXME: HMR時にエラーになるのでwarnにしている。エラーにならないようにする。`みたいな 🙇